src/plot_api/subroutines.js

```diff
@@ -69,16 +69,16 @@ function lsInner(gd) {
 
     function getLinePosition(ax, counterAx, side) {
         var lwHalf = ax._lw / 2;
+        var xshift = ax.position > 0 ? 0 : ax._xshift;
```

Comment:
We get `undefined` and `null` in `xshift` here if you add
```js
console.log(xshift)
```
That could be the reason why we get `NaN` values in SVG.